config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -69,3 +69,4 @@ dashboards:
       alert_mail_to_addresses: patrick.ohly@intel.com
 - name: sig-node-presubmits
 - name: sig-node-image-pushes
+- name: sig-node-release-branches
```

Comment:
let's not have a separate dashboard. release-blocking should suffice

Reply:
Done, moved to release blocking.